src/style.css

```diff
@@ -0,0 +1,59 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars .stars__star {
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--0 .stars__star {
+  background-image: url(./images/star.svg);
+}
+
+.stars--5 .stars__star {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--1 .stars__star:first-child {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--1 .stars__star:not(:first-child) {
+  background-image: url(./images/star.svg);
+}
+
+.stars--2 .stars__star:nth-of-type(-n+2) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--2 .stars__star:not(:nth-of-type(-n+2)) {
+  background-image: url(./images/star.svg);
+}

```

Comment:
try to set active star by default and change where you need on not-active by combining selectors

Reply:
wow! It was unexpected decision for me, but it works ) Thank you )